packages/terra-clinical-result/src/result-name-header-cell/ResultNameHeaderCell.jsx

```diff
@@ -59,16 +70,51 @@ const ResultNameHeaderCell = (props) => {
     customProps.className,
   );
 
+  let resultNameDisplay;
+  if (fullResultName) {
+    resultNameDisplay = (
+      <>
+        <div className={cx('name')} aria-hidden="true">
+          {resultRowIndicators[typeIndicator.toLowerCase()]}
+          {resultName}
+        </div>
+        <VisuallyHiddenText text={fullResultName} />
+      </>
+    );
+  } else {
+    resultNameDisplay = (
+      <div className={cx('name')}>
+        {resultRowIndicators[typeIndicator.toLowerCase()]}
+        {resultName}
+      </div>
+    );
+  }
+
+  let unitDisplay;
+  if (unit) {
+    if (fullUnit) {
+      unitDisplay = (
+        <>
+          <div className={cx('unit')} aria-hidden="true">
+            {unit}
+          </div>
+          <VisuallyHiddenText text={fullUnit} />
+        </>
+      );
+    } else {
+      unitDisplay = <div className={cx('unit')}>{unit}</div>;
+    }
+  } else {
+    unitDisplay = null;
+  }
```

Comment:
We can simply this if ladder by using a ternary operator a second evaluation where it is used (see my next comment)
```suggestion
    const unitDisplay = fullUnit
         ? <>
           <div className={cx('unit')} aria-hidden="true">
             {unit}
           </div>
           <VisuallyHiddenText text={fullUnit} />
         </>
       : <div className={cx('unit')}>{unit}</div>;
```

Reply:
Updated: https://github.com/cerner/terra-clinical/pull/867/commits/361d8279f67328213f80517a2e1ad39a1400c5cf#diff-10a8f15ae7fe78637c3fb3926670dd1d257ecc9c7a37f10b722d3ba6d579ea04R90-R99